src/gradualizer_file_utils.erl

```diff
@@ -56,6 +56,20 @@ epp_parse_file(File, Includes) ->
             Error1
     end.
 
+% epp:open/5 was removed in OTP-24
+% https://github.com/erlang/otp/commit/5281a8c7f77d45a3c36fca9c1a2e4d3812f6fc3d#diff-580a349c49b1d9b5415166e18f5279728d934efe0cebc4ee5a87823055ec3413
+epp_open(File, Fd, StartLocation, Includes) ->
+    code:atomic_load([epp]),
```

Comment:
Nice! This should do. Perhaps slightly cleaner:

```suggestion
    code:ensure_loaded(epp),
```

Reply:
@zuiderkwast yep, that's cleaner.